modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -30,13 +30,20 @@ image::op-pipeline-yaml.png[]
 Configure your pipeline using the *Pipeline Builder*:
 
 .. In the *Name* field, enter a unique name for the pipeline.
-.. In the *Tasks* section, select a task from the *Select Task* list to add a task to the pipeline. In this example, use the *s2i-nodejs* task.
-*** To add sequential tasks to the pipeline, click the plus icon to the right or left of the task, and from the *Select Task* list, select the task you want to add to the pipeline. For this example, use the plus icon to the right of the *s2i-nodejs* task to add an *openshift-client* task.
-*** To add a parallel task to the existing task, click the plus icon displayed next to the task, and from the *Select Task* list, select the parallel task you want to add to the pipeline.
+.. In the *Tasks* section, click *Add task*, search for a task in the quick search, select the required task from the list and click *Add* or *Install and add*. In this example, use the *s2i-nodejs* task.
```

Comment:
The first sentence seems to be quite complex, in the sense that it contains several sequential actions. Can we break it down, may be to sub-bullets?